parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/BlockMetaData.java

```diff
@@ -33,6 +33,7 @@
   private long totalByteSize;
   private String path;
   private int ordinal;
+  private long rowIndexOffset;
```

Comment:
In the following toString(), it should be added too.

Reply:
done.